CHANGELOG.md

```diff
@@ -5,6 +5,95 @@ Documentation of all notable changes to the **Fizzy** project.
+keccak256/512_bytes_rounds_1          105       100        96       162       416        18
+keccak256/512_bytes_rounds_16        1548       100        96       162       418        18
+memset/256_bytes                        7       100        96       155       509        28
+memset/60000_bytes                   1623       100        95       155       507        28
+mul256_opt0/input0                     29       100        91       152       545        22
+mul256_opt0/input1                     29       100        89       152       543        22
+ramanujan_pi/33_runs                  138       100        91       163       483        22
+sha1/512_bytes_rounds_1                94       100        97       181       563        29
+sha1/512_bytes_rounds_16             1318       100        96       179       566        29
+sha256/512_bytes_rounds_1              96       100        98       186       539        30
+sha256/512_bytes_rounds_16           1326       100        97       186       540        30
+taylor_pi/pi_1000000_runs           41668       100        96       154       465        29
+```
+
+### Added
+
+- `fizzy-testfloat` tool to check floating-point instructions implementation against Berkeley TestFloat test suite.
+  [#511](https://github.com/wasmx/fizzy/pull/511)
+- `instantiate` function now has `memory_pages_limit` parameter to configure hard limit on memory allocation.
+  `fizzy-spectests` uses the required 4 GB memory limit now.
+  [#515](https://github.com/wasmx/fizzy/pull/515)
+- `fizzy-spectests` tool can now run tests form a single `*.wast` file. [#525](https://github.com/wasmx/fizzy/pull/525)
+- Use the `shellcheck` tool on CI. [#537](https://github.com/wasmx/fizzy/pull/537)
+- There is now `lib/fizzy/cxx` directory with substitute implementations of C++20 standard library features, which Fizzy
+  uses when compiled with pre-C++20 standard (default). It contains now `span` and `bit_cast` utilities.
+  [#535](https://github.com/wasmx/fizzy/pull/535) [#541](https://github.com/wasmx/fizzy/pull/541)
+
+### Changed
+
+- Optimization in `execute` to do allocate memory only once for all of the arguments, locals and data stack.
+  [#382](https://github.com/wasmx/fizzy/pull/382)
+- Trap handling in `execute` was simplified. [#425](https://github.com/wasmx/fizzy/pull/425)
+- `fizzy-bench` now uses WABT version 1.0.19. [#443](https://github.com/wasmx/fizzy/pull/443)
+- New simpler implementation of `f32.nearest` and `f64.nearest`. [#504](https://github.com/wasmx/fizzy/pull/504)
+- `execute.hpp` now includes declarations only for `execute` functions. Other functions of C++ API (`instantiate`,
+  `resolve_imported_functions`, exports access) were moved to `instantiate.hpp`.
+  [#518](https://github.com/wasmx/fizzy/pull/518)
+- Compiler warning settings now enable more strict checks. [#508](https://github.com/wasmx/fizzy/pull/508)
+- Parsing module with incorrect memory index now throws `validation_error` instead of `parser_error`.
```

Comment:
```suggestion
- Parsing module with incorrect memory/table index now throws `validation_error` instead of `parser_error`.
```

Reply:
Actually this PR is not merged, will remove the line.